Explorer/Assets/Scripts/Global/ComponentsContainer.cs

```diff
@@ -78,6 +78,9 @@ public static ComponentsContainer Create()
                .Add(SDKComponentBuilder<PBAvatarShape>.Create(ComponentID.AVATAR_SHAPE).AsProtobufComponent())
                .Add(SDKComponentBuilder<PBAudioSource>.Create(ComponentID.AUDIO_SOURCE).AsProtobufComponent())
                .Add(SDKComponentBuilder<PBAudioStream>.Create(ComponentID.AUDIO_STREAM).AsProtobufComponent())
+               .Add(SDKComponentBuilder<PBTween>.Create(ComponentID.TWEEN).AsProtobufComponent())
+               .Add(SDKComponentBuilder<PBTweenState>.Create(ComponentID.TWEEN_STATE).AsProtobufComponent())
+               .Add(SDKComponentBuilder<PBTweenSequence>.Create(ComponentID.TWEEN_SEQUENCE).AsProtobufComponent())
```

Comment:
Let's remove this one since we are not going to use it for now

Reply:
done!